search/search.go

```diff
@@ -69,3 +69,16 @@ func MongoDBUsers(a []*opsmngr.MongoDBUser, f func(*opsmngr.MongoDBUser) bool) (
 	}
 	return len(a), false
 }
+
+// MongoDBIndexes return the smallest index i
+// in [0, n) at which f(i) is true, assuming that on the range [0, n),
+// f(i) == true implies f(i+1) == true.
+// returns the first true index. If there is no such index, MongoDBUsers returns n and false
+func MongoDBIndexes(a []*opsmngr.IndexConfigs, f func(configs *opsmngr.IndexConfigs) bool) (int, bool) {
```

Comment:
Thanks for adding this but for future reference if you don't need it don't create it.
(It's ok to keep this for now but I don't think we'll ever need something like this)

Reply:
I am using it in mongocli 😢

Reply:
Ok in that case, please add an example as well. https://github.com/mongodb/go-client-mongodb-ops-manager/blob/master/search/example_search_test.go

Reply:
It should be already there

Reply:
you added a value to the data used for the examples but not an example, please review that file to check how to create an example

Reply:
Also I just reviewed your usage of this function in mcli, any reason why you don't do that in `AddIndexConfig` here directly?